WAW.API/Chat/Resources/MessageRequest.cs

```diff
@@ -0,0 +1,22 @@
+using System.ComponentModel.DataAnnotations;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace WAW.API.Chat.Resources;
+
+public class MessageRequest {
+  [SwaggerSchema("User identifier", Nullable = false)]
+  [Required]
+  public long UserId { get; set; }
```

Comment:
The `UserId` shouldn't be user-input. It should be auto populated from the current logged in User. This should be done at the controller level.

If you allow `UserId` to be user-input (even if it's just sent from the client and not an actual form field in the frontend), anyone would be able to impersonate any other user by just setting `UserId` to the ID of any other person. That's a **huge** security risk.

Actually, there's no way to get the currently logged in user as of right now, so just don't include this in the request resource. The population of the UserId will be done later in the controller once we have the identity system.